### ISSUE TITLE
[Bug]: dwm title bar showing "broken"

### ISSUE THREAD

**Issue by talal12bfk** [termux/termux-packages#12461]
### Problem description

when i install dwm and run it,
it shows the text "broken" in the title bar. I suspect the problem being the missing stdarg.h header file because when i opened the source code of dwm and cross referenced the library being included i didnt find it, the same thing happens with the "tabbed" package which uses the same library. if i build it myself it stays the same.
![Screenshot_20221021-142004](https://user-images.githubusercontent.com/85613292/197185035-c5503f7d-0a3f-45e5-9568-891e8c51aa46.jpg)

### What steps will reproduce the bug?

just start dwm

### System information

termux-info:

```
Termux Variables:
TERMUX_APK_RELEASE=GITHUB
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=19423
TERMUX_IS_DEBUGGABLE_BUILD=1
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.dev/apt/termux-main/ stable main
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.dev/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
termux-tools version:
1.30.2
Android version:
8.0.0
Kernel build information:
Linux localhost 3.18.66-g6a5a99e #1 SMP PREEMPT Mon May 6 16:45:21 CST 2019 aarch64 Android
Device manufacturer:
HUAWEI
Device model:
LDN-L21
```

**Comment by xtkoba** [comment]
Not always for me. In my case "aterm" is shown in the title bar when `aterm` is invoked. "Event Tester" for `xev`.  But when I invoke `epiphany` then "broken" is shown. I wonder if this is related to broken locale support in Xlib (#13362).

**Comment by xtkoba** [comment]
Should be fixed with `libx11` version `1.8.4-1`. Thanks.
